GameDesign/Assets/Scripts/MonoBehaviours/CustomerMonoBehavior.cs

```diff
@@ -71,4 +96,52 @@ bool hasMask()
     {
         return true;
     }
+
```

Comment:
can we add a new state called walking. or neutral or something like that? 
so the mask and un mask are for the transitions when costumers add the masks to themselves. and walking would be the normal behavior.
so `walking` would have transitions `to and from` `masked` and `unmasked`. then from walking we can move to other states,.

Reply:
ok